paystell-frontend/src/components/transaction/AmountInput.tsx

```
@@ -0,0 +1,292 @@
+  inputClassName,
+  error,
+  disabled = false,
+  xlmPrice: propXlmPrice,
+  xlmBalance: propXlmBalance,
+  ...props
+}: AmountInputProps) {
+  // Try to read from Stellar context; fall back to props when not mounted
+  let balances: { asset_type: string; balance: string }[] = [];
+  let hookXlmPrice: number | undefined;
+  try {
+    const { state } = useStellar();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)